pages/_app.js

```diff
@@ -4,6 +4,9 @@ import Layout from "@/components/Layout/Layout";
 import styled from "styled-components";
 import { useEffect, useState } from "react";
 import { FavouritesProvider } from "../components/Favourites/FavouritesContext.js";
+import { CommentsProvider } from "../components/CommentsList/CommentsContext";
```

Comment:
Error: Module not found: Can't resolve '../components/CommentsList/CommentsContext' -> can fix this to CommentsContext.js